docs/src/main/asciidoc/security-jdbc-guide.adoc

```diff
@@ -254,7 +254,7 @@ Finally, using the user `user` works and the security context contains the princ
 curl -i -X GET -u user:user http://localhost:8080/api/users/me
 HTTP/1.1 200 OK
 Content-Length: 4
-Content-Type: text/plain;charset=UTF-8
+Content-Type: application/json
```

Comment:
This was an intentional change from me. It should now be text plain.

See https://github.com/quarkusio/quarkus-quickstarts/blob/master/elytron-security-jdbc/src/main/java/org/acme/elytron/security/jdbc/UserResource.java#L17 .

Could you remove this commit?